docs/advanced-features/custom-server.md

```diff
@@ -46,9 +48,9 @@ app.prepare().then(() => {
     } else {
       handle(req, res, parsedUrl)
     }
-  }).listen(3000, (err) => {
+  }).listen(port, (err) => {
     if (err) throw err
-    console.log('> Ready on http://localhost:3000')
+    console.log(`> Ready on http://localhost:${port}`)
```

Comment:
How about a variable for hostname too

Reply:
Added in https://github.com/vercel/next.js/pull/33744/commits/68e0c604b4c08cae8c66213b679e421daf290e4a and https://github.com/vercel/next.js/pull/33744/commits/c5610932213080724bdfcadac8ce75e3417d8e5c